SwiftCompilerSources/Sources/Optimizer/PassManager/Context.swift

```diff
@@ -256,6 +256,22 @@ struct FunctionPassContext : MutatingContext {
     return false
   }
 
+  func specializeVTable(for type: Type, in function: Function) -> VTable? {
+    guard let vtablePtr = _bridged.specializeVTableForType(type.bridged, function.bridged) else {
+      return nil
+    }
+    notifyInstructionsChanged()
+    return VTable(bridged: BridgedVTable(vTable: vtablePtr))
```

Comment:
Annoying nit: shouldn't the bridging logic be encapsulated in the overlay? So `specializeVTableForType` would just return a `VTable`?

Reply:
Sorry I didn't see what file this was.

Reply:
Discussed offline: This is already in "an overlay" (Context), not in the pass directly, so it sounds fine.